### PR TITLE
Docs: add note and advice about .disabled and keyboard focus

### PR DESCRIPTION
**Pull request by patrickhlauke**
/cc @cvrebert

### CODE REVIEW

File: docs/components/pagination.md [hunk 1]
```diff
@@ -43,11 +43,18 @@ Simple pagination inspired by Rdio, great for apps and search results. The large
 
 Links are customizable for different circumstances. Use `.disabled` for unclickable links and `.active` to indicate the current page.
 
+{% callout warning %}
+#### Disabled links and keyboard access
+
+Although links styled as `.disabled` prevent mouse/touch click functionality, they will still be focusable with the keyboard. To prevent this, include `tabindex="-1"` on the link, or use a static, non-focusable element instead.
```
Comment (cvrebert) [inline docs/components/pagination.md]: Given that we already say that  `pointer-events: none` isn't reliable with regard to browser compatibility, and might be removing it, I wouldn't say that they "prevent mouse/touch click functionality".

Comment (patrickhlauke) [inline docs/components/pagination.md]: If we were to remove `pointer-events:none`, then we should probably consider dropping the use of `<a href="..." class="disabled">` altogether and skip straight to swapping out the `<a>` with a `<span>` or similar, no?

Comment (patrickhlauke) [inline docs/components/pagination.md]: Incidentally, I've had a quick look through the docs, but can't find where we already say anything about `pointer-events:none`. Am I missing it somewhere?

Comment (cvrebert) [inline docs/components/pagination.md]: > skip straight to swapping out the `<a>` with a `<span>` or similar, no?

While we should definitely bring up `<span>`, I suspect there are cases where it won't be a feasible option so we should still cover `<a>`

File: docs/components/pagination.md [hunk 2]
```diff
@@ -43,11 +43,18 @@ Simple pagination inspired by Rdio, great for apps and search results. The large
 
 Links are customizable for different circumstances. Use `.disabled` for unclickable links and `.active` to indicate the current page.
 
+{% callout warning %}
+#### Link functionality caveat
+
+The `.disabled` class uses `pointer-events: none` to try to disable the link functionality of `<a>`s, but that CSS property is not yet standardized and isn't fully supported in Internet Explorer 11\. In addition, even in browsers that do support `pointer-events: none`, keyboard navigation remains unaffected, meaning that sighted keyboard users and users of assistive technologies will still be able to activate these links. So to be safe, add a `tabindex="-1"` attribute on these links (to prevent them from receiving keyboard focus) and use custom JavaScript to disable their functionality.
```
Comment (cvrebert) [inline docs/components/pagination.md]: Extraneous backslash after "11"

File: docs/components/buttons.md [hunk 3]
```diff
@@ -125,7 +125,7 @@ As `<a>` elements don't support the `disabled` attribute, you must add the `.dis
 {% callout warning %}
 #### Link functionality caveat
 
-This class uses `pointer-events: none` to try to disable the link functionality of `<a>`s, but that CSS property is not yet standardized and isn't fully supported in Opera 18 and below, or in Internet Explorer 11\. In addition, even in browsers that do support `pointer-events: none`, keyboard navigation remains unaffected, meaning that sighted keyboard users and users of assistive technologies will still be able to activate these links. So to be safe, use custom JavaScript to disable such links.
+The `.disabled` class uses `pointer-events: none` to try to disable the link functionality of `<a>`s, but that CSS property is not yet standardized and isn't fully supported in Internet Explorer 11\. In addition, even in browsers that do support `pointer-events: none`, keyboard navigation remains unaffected, meaning that sighted keyboard users and users of assistive technologies will still be able to activate these links. So to be safe, add a `tabindex="-1"` attribute on these links (to prevent them from receiving keyboard focus) and use custom JavaScript to disable their functionality.
```
Comment (cvrebert) [inline docs/components/buttons.md]: Interestingly, caniuse now says that `pointer-events` is supported in IE11 (http://caniuse.com/#feat=pointer-events ), which seems to agree with my testing.

Comment (patrickhlauke) [inline docs/components/buttons.md]: Testing this it does indeed seem that IE11 supports `pointer-events:none` just fine. Removing that piece of advice.

**Comment by patrickhlauke** [comment]
x-ref also https://github.com/twbs/bootstrap/pull/18699

**Pull request merged**